lightning/src/chain/keysinterface.rs

```diff
@@ -325,7 +337,12 @@ pub trait ChannelKeys : Send+Clone {
 	/// We bind holder_selected_contest_delay late here for API convenience.
 	///
 	/// Will be called before any signatures are applied.
-	fn on_accept(&mut self, channel_points: &ChannelPublicKeys, counterparty_selected_contest_delay: u16, holder_selected_contest_delay: u16);
+	fn on_accept(&mut self, channel_points: &ChannelPublicKeys, counterparty_selected_contest_delay: u16, holder_selected_contest_delay: u16, is_outbound: bool);
```

Comment:
Actually, I believe this can be merged into `on_funding_created` because we don't need this information until then.  No signatures can happen until we know the funding outpoint anyway.

Reply:
merged